packages/autorest.python/autorest/m4reformatter/__init__.py

```diff
@@ -577,6 +577,9 @@ def update_lro_initial_operation(self, initial_operation: Dict[str, Any]):
         initial_operation["name"] = f"_{initial_operation['name']}_initial"
         initial_operation["isLroInitialOperation"] = True
         initial_operation["wantTracing"] = False
+        if self.legacy:
+            # legacy will use msrest models, which may meet https://github.com/Azure/autorest.python/issues/2428 for lro
+            initial_operation["exposeStreamKeyword"] = True
```

Comment:
Only msrest model will meet the issue so I think there is no need to influence version-tolerant mode. Now only swagger is permitted for non-version-tolerant so it is better to add it here.

Reply:
i think it'd be better to do it for all generations of LRO, because we don't want too much "if else" code. Ultimately, we are saving unnecessary deserialization here, so I think it's good to have this be fixed everywhere

Reply:
Make sense.